components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -274,6 +290,13 @@ void run(std::mt19937_64& engine)
     // Compute vmr from mmr and vice versa
     PF::calculate_vmr_from_mmr(team,h2o_mol,qv,mmr_for_testing,vmr);
     PF::calculate_mmr_from_vmr(team,h2o_mol,qv,vmr,mmr);
+
+    // Compute wetmmr assuming mmr is dry
+    PF::calculate_wetmmr_from_drymmr(team,mmr,qv,wetmmr);//get wetmmr from mmr, assuming mmr is drymmr
```

Comment:
These tests look great, and thanks for catching my comment error in the vmr,mmr portion.

If you follow the example of vmr->mmr and vice versa there is also a few checks below where we deep copy to host the output from these operations on the view and do some checks, like not negative.  Could you add that in too?

You could also follow the vmr/mmr example and check that `wetmmr` is approx_equal to `mmr` for all data in the view.  See line 342.

Reply:
I was converting it back to `mmr` here as well which is tested downstream so I didn't add the code for wet and dry mmr. More testing is always better, so I followed your suggestion to add checks for both dry and wet mmr. I inserted a loop for assigning `mmr` to `drymmr`. Let me know if there is a better way to handle this assignment.